tests/unit/api/test_api.py

```diff
@@ -43,6 +43,42 @@
 
 class TestInitialization(DatadogAPINoInitialization):
 
+    def test_default_settings_set(self):
+        """
+        Test all the default setting are properly set before calling initialize
+        """
+        from datadog.api import (
```

Comment:
I am not really convinced by the usefulness of this test. It won't prevent anyone to not add settings in the init later. What do you think?